src/main/flight/pid.c

```diff
@@ -388,6 +388,17 @@ void pidUpdateTpaFactor(float throttle)
 
     DEBUG_SET(DEBUG_TPA, 0, lrintf(tpaFactor * 1000));
     pidRuntime.tpaFactor = tpaFactor;
+    
+#ifdef USE_WING
+    switch (currentPidProfile->yaw_type) {
+    case YAW_TYPE_DIFF_THRUST:
+        pidRuntime.tpaFactorYaw = getTpaFactorClassic(tpaArgument);
+        break;
+    case YAW_TYPE_RUDDER:
+    default:
+        pidRuntime.tpaFactorYaw = pidRuntime.tpaFactor;
+    }
```

Comment:
```suggestion
        break;
    }
```

Reply:
Last case does not need a break, but from K&R:

_As a matter of good form, put a break after the last case (the default here) even though it's logically unnecessary. Some day when another case gets added at the end, this bit of defensive programming will save you._

Reply:
we gonna call it Karate break :)

Reply:
In this case it is handling YAW_TYPE_RUDDER to, so it is a bit reasonable

Reply:
(and on the other side, gcc has fallthrough warning enabled, so it will catch K&R mentioned problem automatically)